packages/utah-design-system/src/contexts/FirebaseAuthProvider.tsx

```diff
@@ -57,6 +58,18 @@ export const FirebaseAuthProvider = (props: FirebaseAuthProviderProps) => {
   const login = () => signInWithPopup(sdk, provider);
   const logout = () => signOut(sdk);
 
+  useEffect(() => {
+    if (app && import.meta.env.DEV) {
+      const auth = getAuth(app);
+      if (!auth.emulatorConfig) {
+        console.log('connecting to auth emulator');
+        connectAuthEmulator(auth, 'http://127.0.0.1:9099', {
+          disableWarnings: true,
+        });
```

Comment:
**question**: do you find that this throws if it's already connected and is run multiple times?